pkg/virt-controller/services/template_test.go

```diff
@@ -1964,69 +1945,45 @@ var _ = Describe("Template", func() {
 				}
 			})
 			It("should add 1G of memory overhead", func() {
-				sriovInterface := v1.InterfaceSRIOV{}
-				domain := v1.DomainSpec{
-					Resources: v1.ResourceRequirements{
-						Requests: kubev1.ResourceList{
-							kubev1.ResourceMemory: resource.MustParse("1G"),
-						},
-					},
-				}
-				domain.Devices.Interfaces = []v1.Interface{{Name: "testnet", InterfaceBindingMethod: v1.InterfaceBindingMethod{SRIOV: &sriovInterface}}}
-				vmi := v1.VirtualMachineInstance{
-					ObjectMeta: metav1.ObjectMeta{
-						Name: "testvmi", Namespace: "default", UID: "1234",
+				vmi := newVMIWithSriovInterface("testvmi", "1234")
+				vmi.Spec.Domain.Resources = v1.ResourceRequirements{
+					Requests: kubev1.ResourceList{
+						kubev1.ResourceMemory: resource.MustParse("1G"),
 					},
-					Spec: v1.VirtualMachineInstanceSpec{Domain: domain},
 				}
+				domain := vmi.Spec.Domain
 
-				pod, err := svc.RenderLaunchManifest(&vmi)
+				pod, err := svc.RenderLaunchManifest(vmi)
 				Expect(err).ToNot(HaveOccurred())
 				expectedMemory := resource.NewScaledQuantity(0, resource.Kilo)
-				expectedMemory.Add(*getMemoryOverhead(&vmi))
+				expectedMemory.Add(*getMemoryOverhead(vmi))
 				expectedMemory.Add(*domain.Resources.Requests.Memory())
 				Expect(pod.Spec.Containers[0].Resources.Requests.Memory().Value()).To(Equal(expectedMemory.Value()))
 			})
 			It("should still add memory overhead for 1 core if cpu topology wasn't provided", func() {
-				domain := v1.DomainSpec{
-					Resources: v1.ResourceRequirements{
-						Requests: kubev1.ResourceList{
-							kubev1.ResourceMemory: resource.MustParse("512Mi"),
-							kubev1.ResourceCPU:    resource.MustParse("150m"),
-						},
+				requests := v1.ResourceRequirements{
```

Comment:
`requests`? Maybe you meant `requirements` ?

Reply:
Yes I will change it